mln/services/message.py

```diff
@@ -65,6 +65,9 @@ def send_message(message, attachment):
 			for attachment in trigger.response.attachments.all(): 
 				response.attachments.create(item_id=attachment.item_id, qty=attachment.qty)
 			break
+		else:  # networker doesn't have a trigger for this message
+			body = MessageBody.objects.filter(text=MLNError.I_DONT_GET_IT).first()
+			Message.objects.create(sender=message.recipient, recipient=message.sender, body=body)

```

Comment:
The "I don't get it" fallback response should return any attachments from the user back to the user as attachments.